content/_changelogs/11.0.0.md

```diff
@@ -1,17 +1,100 @@
 ## 11.0.0
 
-_Released MM/DD/YYYY_
+_Released 11/08/2022_
 
 **Summary:**
 
+Component Testing is now generally available for projects using React, Next.js,
+Angular, and Vue!
+
+Component tests allow you to see and test your application's components in a
+real browser as you work. You can use your favorite Cypress commands and
+features to develop your components without running your whole app. Learn more
+in our [blog](https://on.cypress.io/cypress-11-release) post.
+
+We have also massively improved our startup performance by shipping a snapshot
+of our binary instead of the source files. Results will vary based on your
+situation, but we saw up to 84% faster startup times!
+
 **Breaking Changes:**
 
-**Deprecations:**
+**Read our [Migration Guide](/guides/references/migration-guide) which explains
+the breaking changes in more detail.**
+
+- Removed `cssFile`, `cssFiles`, `style`, `styles`, `stylesheet`, and
+  `stylesheets` from the `options` argument to the `cy.mount()` function. We
+  recommend writing test-specific styles in a separate `css` file you import in
+  your test, or in your `supportFile`. Addresses
+  [24328](https://github.com/cypress-io/cypress/issues/24328)
+- Subsequent `cy.mount` calls within the same test will remove the last mounted
+  component from the DOM. Addresses
+  [#24329](https://github.com/cypress-io/cypress/issues/24329).
+- `mountHook` from `cypress/react` has been removed. We recommend replacing it
+  with `mount` and a component. Addresses
+  [24328](https://github.com/cypress-io/cypress/issues/24328)
+- `unmount` from `cypress/react` has been removed. We recommend using the API
+  React provides for unmounting components,
+  [unmountComponentAtNode](https://reactjs.org/docs/react-dom.html#unmountcomponentatnode).
+  Addresses [24328](https://github.com/cypress-io/cypress/issues/24328)
+- `mountCallback` from `cypress/vue` has been removed. We recommend using
+  `mount`. Addresses [24328](https://github.com/cypress-io/cypress/issues/24328)
+- When providing an inline `viteConfig` inside of `cypress.config`, any
+  `vite.config.js` file is not automatically merged. Addresses
+  [#24133](https://github.com/cypress-io/cypress/issues/24133)
+- Angular providers passed as part of the mounting options will be assigned at
+  the module level using the `TestBed.configureTestingModule` API. This means
+  that module-level providers (resolved from imports or
+  `@Injectable({ providedIn: 'root' })` can be overridden, but providers
+  specified in `@Component({ providers: [...] })` will not be overridden when
+  using `cy.mount(MyComponent, { providers: [...] })`. To override
+  component-level providers, use the `TestBed.overrideComponent` API. Addresses
```

Comment:
These look good, we could put a link to the specific part of the blog post for each breaking change (each one has it's own section). Not sure if this is necessary or not, but if someone misses the initial link at the top, the migration path might not be clear.

Reply:
Added link to the migration guide sections. [7ae5a18](https://github.com/cypress-io/cypress-documentation/pull/4844/commits/7ae5a185f240140ad11181eb03b47dcc14e15df5)